lib/megaLogger.js

```diff
@@ -132,23 +136,21 @@
             'LOG': '#000000'
         },
         'dateFormatter': function(d) {
-            return d.getFullYear() + "-" + d.getMonth() + "-" + d.getDate() + " " + d.getHours() + ":" + d.getMinutes() + ":" + d.getSeconds() + "." + d.getMilliseconds();
+            return d.toISOString();
         },
-        'transport': function() {
-            var level = Array.prototype.slice.call(arguments, 0);
-            var args = level.splice(1);
+        'transport': function(level, args) {
```

Comment:
:+1:

originally, i had the idea of the 'transport' func to get called with some a bit more dynamic arguments, but ... it did not made sense after a while, so nice catch :)